web/src/pages/dashboard/Dashboard.tsx

```diff
@@ -1,11 +1,12 @@
 import React, { useContext } from "react";
 import { Navigate } from "react-router-dom";
 import { AuthContext } from "../../context/auth.context";
+import Layout from "../../components/layout/Layout";
 
 function HomePage() {
-	const { authState } = useContext(AuthContext);
-	if (!authState.isLoggedIn) return <Navigate to="/login" />
-	return `Welcome to your Dashboard, ${authState.firstName}`;
+  const { authState } = useContext(AuthContext);
+  if (!authState.isLoggedIn) return <Navigate to="/login" />;
+  return <Layout>`Welcome to your Dashboard, ${authState.firstName}`</Layout>;
```

Comment:
S: Remove backticks, and `$`, so that the `Welcome` message is treated as text, and only the `{authState.firstName}` is used as a variable reference.

Reply:
Commit: b0a39710a9dd49a1ffbe3b3c06c12cb371898c25